### PR TITLE
[CI] - coveralls update on coverage job

### PR DESCRIPTION
**Pull request by GuyAfik**
## Description
coveralls updates

### CODE REVIEW

File: .github/workflows/on-push.yml [hunk 1]
```diff
@@ -205,13 +205,13 @@ jobs:
           coverage combine **/.coverage
           coverage report
           coverage xml
-      - name: Coveralls
+      - name: Coveralls-Action
         uses: coverallsapp/github-action@v2
-      - name: Coveralls v2
+      - name: Coveralls-Comment
```
Comment (ilaner) [inline .github/workflows/on-push.yml:210]:
```suggestion
      - name: Coveralls-Comment
         env:
             COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
            
```

Comment (ilaner) [reply .github/workflows/on-push.yml:210]: and then you don't need to change the script at all

**Review by ilaner — CHANGES_REQUESTED** [review]
see comment

File: .github/workflows/on-push.yml (hunk 1 above)
Comment (GuyAfik) [reply .github/workflows/on-push.yml:210]: good idea, done

**Review by tkatzir — APPROVED** [review]
Great! Approved!

**Pull request merged**